config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  delete 'news/:id', to: 'news#destroy'
```

Comment:
Convendría implementar directamente:
```suggestion
  resources :news, only: %i[destroy]
```